### DISCUSSION TITLE
Pull or download any changes

### DISCUSSION THREAD

**Discussion by Yohn** [ObsidianPublisher/obsidian-github-publisher#353] [Q&A — Plugin]
Hello,

I was previously tinkering around with this when it was github publisher, but was never able to get it to work on my desktop or mobile device, and I just started messing with it again today, took a minute to realize the name changed but once I figured that out I was able to get both desktop and mobile connected to one of my private repos here on github.

What I'd like to do is keep both desktop and mobile in sync, so if say I update a note on mobile, I can push to the repo and see the change in my repo, but then I'd like to download that change to my desktop to see the change there. This way the notes are viewable, by the most recent updates on both desktop and mobile. 

Is there a way to pull the changes in my repo to the desktop or mobile app when there are any changes between the 2?

**Comment by ooker777** [comment]
This sounds like you are only use github as a place to sync your vault between devices, not as a step to publish website? If that's the case then this plugin is not what you are looking for. The [Git plugin](https://publish.obsidian.md/git-doc/Start+here "Start here - Git Documentation - Obsidian Publish") is what you are looking for. To be specific, use the auto-pull feature of it.
